tools/Environments/DevHome.Environments/Views/LandingPage.xaml

```diff
@@ -79,7 +79,8 @@
                 <Grid>
                     <Button
                         Style="{StaticResource HorizontalThreeDotsStyle}"
-                        AutomationProperties.Name="{x:Bind MoreOptionsButtonName}">
+                        AutomationProperties.Name="{x:Bind MoreOptionsButtonName}"

```

Comment:
Can `x:UID="MoreOptionsButton"` be used here as well?

Reply:
@AmelBawa-msft So, the standard we should follow everywhere in the project is to use the x:UID?

Some parts of DevHome are like this and others are direct binding.

Reply:
Not sure there' s a particular standard, but we should try to be consistent. The way I think about it, if the property is related to localization, I'd consider defining them in the resource file. If the property is bound to a model, view model or other elements, then set it from XAML. That said, there are many ways to do this, we can even bind the property to another one for the same control. In this PR, since we used `x:UID` above, my nit suggestion is to apply the same solution for the same problem.

Reply:
My understanding is that x:Uid is always preferred and we only use binding when something prevents us from using x:Uid (a specific type of variable inside the string, needing to change the string dynamically, things like that).

Reply:
@guimafelipe 
> Some parts of DevHome are like this and others are direct binding.
Those parts are mine.  Especially in AddRepoDialog and RepoConfig.  I'm working on moving those over.

Reply:
Since changing this binding does not resolve the original bug, I'm going to change it in another PR.